plugins/inputs/snmp/snmp.go

```diff
@@ -34,25 +33,20 @@ const sampleConfig = `
   ##            agents = ["tcp://127.0.0.1:161"]
   ##            agents = ["udp4://v4only-snmp-agent"]
   agents = ["udp://127.0.0.1:161"]
-
   ## Timeout for each request.
   # timeout = "5s"
-
   ## SNMP version; can be 1, 2, or 3.
   # version = 2
-
+  ## Path to mib files
+  # path = ["/usr/share/snmp/mibs"]
   ## Agent host tag; the tag used to reference the source host
   # agent_host_tag = "agent_host"
-
   ## SNMP community string.
   # community = "public"
-
   ## Number of retries to attempt.
   # retries = 3
-
   ## The GETBULK max-repetitions parameter.
   # max_repetitions = 10
-
```

Comment:
Is this enforced by the linter? If so, please also reflect in Readme file. If not, please revert.